src/Microsoft.Dnx.TestHost/ProjectTestHostServices.cs

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Linq;
+using Microsoft.Dnx.Compilation;
+using Microsoft.Dnx.Runtime.Common.DependencyInjection;
+using Microsoft.Dnx.TestHost.TestAdapter;
+using Microsoft.Dnx.Testing.Abstractions;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Dnx.TestHost
+{
+    internal class ProjectTestHostServices: TestHostServices
+    {
+        public ProjectTestHostServices(
+               Project project,
+               ReportingChannel channel)
```

Comment:
Don't make it harder on the next person who edits this file. Just leave the parameters where VS puts them.